plugins/node/opentelemetry-instrumentation-mongodb/package.json

```diff
@@ -7,7 +7,8 @@
   "repository": "open-telemetry/opentelemetry-js-contrib",
   "scripts": {
     "docker:start": "docker run -e MONGODB_DB=opentelemetry-tests -e MONGODB_PORT=27017 -e MONGODB_HOST=localhost -p 27017:27017 --rm mongo",
-    "test": "nyc ts-mocha --parallel -p tsconfig.json --require '@opentelemetry/contrib-test-utils' 'test/**/*.test.ts'",
+    "test": "nyc ts-mocha -p tsconfig.json --require '@opentelemetry/contrib-test-utils' 'test/**/mongodb-v4.test.ts'",
```

Comment:
this change means we are running PullRequests CI tests only for v4, and stop running v3 tests on PRs.
https://github.com/open-telemetry/opentelemetry-js-contrib/blob/81b3190af64bda14f87c5b0cbd6172bafda26408/package.json#L28
something we might want to consider...

Reply:
That is correct. I can't think about any other option. Suggestions?

Reply:
Not a particularly good one
We can run `tav` here as well, possibly with just 2 versions. But that sounds clumsy and complex.
Let's wait for more minds on this one @Rauno56 @dyladan

Reply:
btw, The tests are only can run locally (we need mongodb server). So anyway i dont think i would change much

Reply:
Yes, I usually spin up a mongo server locally with docker and run the tests against it.
CI tests also start a mongo server [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/81b3190af64bda14f87c5b0cbd6172bafda26408/.github/workflows/unit-test.yml#L27), so the tests can pass.

Reply:
We talked about this in the SIG. We recommend to test the most popular major version (in this case v3) and test other versions with TAV.